.github/workflows/stale.yml

```diff
@@ -0,0 +1,17 @@
+name: "Close stale issues"
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: 'This issue is stale because it has been open many days with no activity. It will be closed soon unless the stale label is removed or a comment is made.'
+        stale-issue-label: 'stale'
+        exempt-issue-label: 'never stale'
```

Comment:
thought I cannot think of anything concrete, can we think about a better single word for this label? like `immortal` , `active`, `never-stale` etc.

Reply:
Sorry, ended up merging without making this change. I'll create the `never stale` label just so that the action doesn't break, but we can rename later.